### ISSUE TITLE
Is there a way to password protect stream?

### ISSUE THREAD

**Issue by rassha** [iizukanao/picam-streamer#18]
Great work... Is there a way to password protect stream (http web page) and/or vlc stream?

**Comment by iizukanao** [comment]
@rassha Yes, you can integrate Basic Auth into your website by using nginx or Apache as a reverse proxy.
